Ambit/Source/Ambit/Mode/ConfigImportExport.cpp

```diff
@@ -940,6 +940,12 @@ void UConfigImportExport::SetMockWriteFile(
     LambdaWriteFileToDisk = std::move(MockFunction);
 };
 
+void UConfigImportExport::SetMockCompressFile(
+    TFunction<FString(const FString& SourceDirectory, const FString& TargetDirectory, const FString& FileName, const FString& TargetPlatform)> MockFunction)
```

Comment:
Has this gone through the ReSharper code formatting? It seems incorrect.

Reply:
Refactored.